ruby-enum.gemspec

```diff
@@ -10,7 +10,7 @@ Gem::Specification.new do |s|
   s.email = 'dblock@dblock.org'
   s.platform = Gem::Platform::RUBY
   s.required_rubygems_version = '>= 1.3.6'
-  s.required_ruby_version = '>= 2.3'
+  s.required_ruby_version = '>= 2.7'
```

Comment:
We could keep the 2.3 compat by separating the gemfile for Rubocop, Danger, etc, like in https://github.com/slack-ruby/slack-ruby-client/blob/master/.github/workflows.

Reply:
Is it worth it?

Reply:
Personally I think it is because it simplifies the overall dependency management by splitting Gemfile(s) - I'd like to be able to upgrade rubocop without upgrading everything for example, but I don't want to create more work for you. Your call!

Reply:
I think the issue is that using Rubocop well really requires a commitment to a minimum Ruby version - and one that's not too far EOLed.  Right now very few Rubocop cops are active, which makes Rubocop kind of minimal value.

Long term I'd suggest moving to all cops enabled, which makes maintaining old Ruby compatibility more challenging.

Reply:
My primary goal with rubocop has always been to have an auto-formatter so we don't have to debate tabs vs. spaces. I agree that many other cops are useful and totally +1 on enabling them all, as long as someone better than me takes care of it :)